.jenkins/cscs-ault/env-hipcc.sh

```diff
@@ -12,7 +12,7 @@ hwloc_version="2.6.0"
 spack_compiler="gcc@${gcc_version}"
 spack_arch="linux-centos8-zen"
 
-spack_spec="pika@main+rocm arch=${spack_arch} %${spack_compiler} malloc=system cxxstd=${cxx_std} ^boost@${boost_version} ^hwloc@${hwloc_version} ^hip@5.2.0"
+spack_spec="pika@main+rocm arch=${spack_arch} %${spack_compiler} malloc=system cxxstd=${cxx_std} ^boost@${boost_version} ^hwloc@${hwloc_version} ^hip@5.2.3"
```

Comment:
Maybe you can move the version out into a `hip_version` variable? Thanks for updating this!